hw/bsp/dialog_da1469x-dk-pro/da1469x_header_tool.py

```diff
@@ -0,0 +1,282 @@
+#! /usr/bin/env python3
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import re
+import click
+import io
+import os.path
+import struct
+import binascii
+import time
+from typing import NamedTuple
+import base64
+
+from cryptography.hazmat.primitives.ciphers import Cipher, algorithms, modes
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+
+from cryptography.hazmat.primitives import hashes
+
+# Add in patch to mcuboot.  This is required to pull in keys module from imgtool
+import sys
+sys.path.append(os.getcwd() + "/repos/mcuboot/scripts/imgtool")
+import keys as keys
+
+class product_header(NamedTuple):
+    ident: bytes
+    active_addr: int
+    update_addr: int
+    cmda_reg: int
+    cmdb_reg: int
+    flash_ident: bytes
+    flash_length: int
+    cmd_seq: bytes
+
+class key_revocation_entry(NamedTuple):
+    key_type: int
+    key_index: int
+
+class revocation_section(NamedTuple):
+    ident: bytes
+    length: int
+
+class device_administration_section(NamedTuple):
+    ident: bytes
+    length: int
+
+class signature_section(NamedTuple):
+    sig_idx: int
+    enc_idx: int
+    nonce: bytes
+    ident: bytes
+    length: int
+    signature: bytes
+
+class security_section(NamedTuple):
+    ident: bytes
+    length: int
+
+class fw_header(NamedTuple):
+    ident: bytes
+    length: int
+    crc: int
+    version: bytes
+    timestamp: int
+    ivt_offset: int
+
+def crc16(data : bytearray, offset , length):
```

Comment:
PEP-8: no space before `:` (`data: xxx`), no space before `,` (`offset, `)

Reply:
fixed.  I fixed a lot of other pep8 issues as well.